File: evidence/generated-systems/golang/fintech-on-golang/fintech-on-golang/internal/model/RiskAssessment.go
package model

import (
    "time"
    "gorm.io/gorm"
)

//==============================================================
// RiskAssessment Declaration
//==============================================================
type RiskAssessment struct {
    gorm.Model
     Score                                                            string
    AssessedAt                                                            time.Time
    ModelVersion                                    string
    Notes                                    string
    ApplicationId         *uint
    Application           *LoanApplication `gorm:"foreignKey:ApplicationId"`
    Decision                      DecisionOutcome

// parent associations as their child

}

File: evidence/generated-systems/django/fintech-on-django/fintechOnDjango/models/RiskAssessment.py
from django.db import models
from fintechOnDjango.models.DecisionOutcome import DecisionOutcome

#======================================================================
# 
# Encapsulates data for model RiskAssessment
#
# @author Harbormaster Dev Team
#
#======================================================================

#======================================================================
# Class RiskAssessment Declaration
#======================================================================
class RiskAssessment (models.Model):

#======================================================================
# attribute declarations
#======================================================================
	score = RiskScore
	assessedAt = DateTime
	modelVersion = models.CharField(max_length=200, null=True)
	notes = models.CharField(max_length=200, null=True)
	application = models.ForeignKey('LoanApplication', on_delete=models.CASCADE, null=True, blank=True, related_name='+')
	decision = models.CharField(max_length=64, null=True, choices=[(tag.name, tag.value) for tag in DecisionOutcome])

#======================================================================
# function declarations
#======================================================================
	def toString(self):
		str = ""
		str = str + self.score
		str = str + self.assessedAt
		str = str + self.modelVersion
		str = str + self.notes
		str = str + self.decision
		return str;
    
	def __str__(self):
		return self.toString();

	def identity(self):
		return "RiskAssessment";
    
	def objectType(self):
		return "RiskAssessment";
